Library/Formula/pilot-link.rb

```diff
@@ -0,0 +1,22 @@
+class PilotLink < Formula
+  homepage "http://www.pilot-link.org/"
+  url "http://downloads.pilot-link.org/pilot-link-0.12.5.tar.gz"
+  version "0.12.5"
+  sha256 "159b84cadf5cdb39e238feaa652e98fc289a5624356a8ddd7f84d6a55eb5c736"
+
+  depends_on "popt" => :build
+  depends_on "libpng12" => :build
```

Comment:
Thanks. Going away for a week, but will come back with an update.

Regards
Jon

On 28. mars 2015 20:32, Dominyk Tiller wrote:
>
> In Library/Formula/pilot-link.rb 
> <https://github.com/Homebrew/homebrew/pull/38120#discussion_r27347676>:
>
> > @@ -0,0 +1,22 @@
> > +class PilotLink < Formula
> > +  homepage "http://www.pilot-link.org/"
> > +  url "http://downloads.pilot-link.org/pilot-link-0.12.5.tar.gz"
> > +  version "0.12.5"
> > +  sha256 "159b84cadf5cdb39e238feaa652e98fc289a5624356a8ddd7f84d6a55eb5c736"
> > +
> > +  depends_on "popt" => :build
> > +  depends_on "libpng12" => :build
>
>     Hmm. Debian has a patch to use modern libpng. Is there a way to
>     use patches?
>
> Yup:
>
> depends_on"libpng"
>
> # Patch does x
> # Upstream bug report link (ideally):
> patchdo
>    url""
>    sha256""
> end
>
> def  install
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Homebrew/homebrew/pull/38120/files#r27347676>.
>